src/components/app/HealthCheckReport/index.tsx

```diff
@@ -38,7 +38,7 @@ const headCells: HeadCell<GenericListItem>[] = [
         format: (item: GenericListItem) => {
             return (
                 <Link
-                    color="inherit"
+                    style={{ color: '#ce93d8' }}
```

Comment:
The link color is not good-looking on the light theme. Blue is better.
![image](https://user-images.githubusercontent.com/46446076/155385962-052233c7-abe3-4b10-9744-87f56b8c2e2b.png)